ACE/ace/Basic_Types.h

```diff
@@ -70,6 +70,12 @@
 #  include <inttypes.h>
 # endif
 
+# if defined (ACE_HAS_CPP11)
+#  include <cstdint>
+#  define ACE_HAS_INT64_T
```

Comment:
@jwillemsen had mentioned adding #defines for the other types as well.  Is that still possible?

Reply:
I haven't thought carefully about those types yet, but I guess it is possible. I'm not sure if we should include those in this PR though.

Reply:
We should do that at the same moment, see https://github.com/DOCGroup/ACE_TAO/pull/1066

Reply:
Ok, I'll add those types.